docs/source/installation.rst

```diff
@@ -10,10 +13,30 @@ environment. E.g. with ``venv``:
    python -m venv ~/.venvs/onyo
    source ~/.venvs/onyo/bin/activate
 
-Clone the repo and install the package.
+Clone the repo and install the package
+**************************************
+
+Run the following from your command line:
 
 .. code::
 
    git clone https://github.com/psyinfra/onyo.git
    cd onyo
    pip install -e .
+
+Install Dependencies
+********************
+
+Additional non-python dependencies need to be installed:
+
+**Debian/Ubuntu**:
+
+.. code:: shell
+
+   apt-get install git tig tree
+
+**macOs**:
```

Comment:
World's tiniest nit, but it slays me: macOs -> macOS

:-D

Reply:
Oops, thanks for catching that!